### PR TITLE
Add June 2021's meeting mins

### LINKED ISSUE #414: Hubs Team Meeting - June 2021
Hello everyone!

Reminder that our next meeting is **Thursday, 17th June, 2021 at 9AM Europe/London** (or [your timezone](https://arewemeetingyet.com/London/2021-06-17/09:00/Hubs-Team-Meeting)).

Agenda - HackMD: https://hackmd.io/@sgibson91/hubs-team-meeting
Video - Google Meet: https://meet.google.com/kxq-hycx-bvt

These calls are open to **anyone**, so please join us! We love seeing new members on the call 😄

**In this meeting, we will also be trialling a speakers list. See issue #411 for context.**

### PR DESCRIPTION
**Pull request by sgibson91**
fixes #414

### CODE REVIEW

File: docs/monthly-meeting/2021-06-17.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+60 second updates on things you have been up to, questions you have, or developments you think people should know about. Please add yourself, and if you do not have an update to share, you can pass.
+
+-
+
+## Reports and celebrations
+
+This is a place to make announcements (without a need for discussion). This is also a great place to give shout-outs to contributors! We'll read through these at the beginning of the meeting.
+
+- Distinguished Jupyter contributors announced: https://blog.jupyter.org/congratulations-distinguished-contributors-149deff3033c
+- Erik & 1.0!
```
Comment (manics) [inline docs/monthly-meeting/2021-06-17.md:40]: Do you know what this is referring to? Z2JH 1.0.0?

**Review by manics — APPROVED** [review]
One minor query but otherwise looks good to merge

File: docs/monthly-meeting/2021-06-17.md (hunk 1 above)
Comment (sgibson91) [reply docs/monthly-meeting/2021-06-17.md:40]: Yes I think so

Comment (sgibson91) [reply docs/monthly-meeting/2021-06-17.md:40]:
```suggestion
- Erik & Z2JH 1.0.0 release!
```

Comment (sgibson91) [reply docs/monthly-meeting/2021-06-17.md]: Added a clarification :D

**Pull request merged**